src/Boards/Board.php

```diff
@@ -140,10 +91,24 @@ public function compare(Board $_board)
      */
     public function setCell($_x, $_y, $_value)
     {
-        if ($_x < 0 || $_y < 0 || $_x >= $this->width || $_y >= $this->height)
+        if ($this->isOutOfBounds($_x, $_y))
             return;
 
-        $this->grid[$_x + 1][$_y + 1] = $_value;
+        $this->grid[$_x + 1][$_y + 1]->setValue($_value);
+    }
+
+    /**
+     * Returns a cell at the given point.
+     * @param int $_x X position of the cell.
+     * @param int $_y Y position of the cell.
+     * @return Field|null The cell or null pointer on invalid coordinates.
+     */
+    public function getCell(int $_x, int $_y): ?Field

```

Comment:
I think it would be better to rename this method to getField() (or even field()) because "Cell" is not a word that is used elsewhere. (Or rename the Field class to Cell 😄)